api/src/main/java/org/openmrs/api/impl/ConceptServiceImpl.java

```diff
@@ -1977,4 +1977,46 @@ public boolean hasAnyConceptAttribute(ConceptAttributeType conceptAttributeType)
 		}
 		return mappedClasses;
 	}
+
+	public Concept getConceptByRef(String conceptRef) throws APIException {
```

Comment:
@jnsereko I would keep the naming to:
```java
public Concept getConcept(String conceptRef)

```

Or anything else better?

Reply:
the method `getConcept(String)` already exists in the class it results into a conflict, unless when i delete the aready existing `getConcept(String)` method.

Reply:
how is `getConceptByNameIdOrMap(String)`?